borsh/src/schema.rs

```diff
@@ -401,6 +402,48 @@ impl BorshSchema for str {
     }
 }
 
+#[cfg(feature = "ascii")]
+impl BorshSchema for ascii::AsciiString {
+    #[inline]
+    fn add_definitions_recursively(definitions: &mut BTreeMap<Declaration, Definition>) {
+        ascii::AsciiStr::add_definitions_recursively(definitions);
+    }
+    #[inline]
+    fn declaration() -> Declaration {
+        ascii::AsciiStr::declaration()
+    }
+}
+
+#[cfg(feature = "ascii")]
+impl BorshSchema for ascii::AsciiStr {
+    #[inline]
+    fn add_definitions_recursively(definitions: &mut BTreeMap<Declaration, Definition>) {
+        let definition = Definition::Sequence {
+            length_width: Definition::DEFAULT_LENGTH_WIDTH,
+            length_range: Definition::DEFAULT_LENGTH_RANGE,
+            elements: u8::declaration(),
+        };
+        add_definition(Self::declaration(), definition, definitions);
+        u8::add_definitions_recursively(definitions);
+    }
+    #[inline]
+    fn declaration() -> Declaration {
+        "AsciiString".into()
+    }
+}
+
+#[cfg(feature = "ascii")]
+impl BorshSchema for ascii::AsciiChar {
+    #[inline]
+    fn add_definitions_recursively(definitions: &mut BTreeMap<Declaration, Definition>) {
+        add_definition(Self::declaration(), Definition::Primitive(1), definitions);
+    }
+    #[inline]
+    fn declaration() -> Declaration {
+        "AsciiChar".into()
+    }
+}
```

Comment:
please add tests for schema generated for these 3 types right after https://github.com/near/borsh-rs/blob/master/borsh/src/schema.rs#L1025-L1058